site/docs/v1/tech/apis/scim/_scim-user-response-body.adoc

```diff
@@ -0,0 +1,72 @@
+Because SCIM schemas are completely customizable, the SCIM response returned is generated from the result of the SCIM User object populated by your outgoing response lambda.
+
+The following is an example of a typical SCIM User response body that your SCIM client might expect. However, your outgoing response lambda could generate a completely different SCIM schema is desired.
+
+This is taken from the SCIM Schema RFC describing a SCIM User schema. For the full specification you can find the RFC here:
+
+- https://datatracker.ietf.org/doc/html/rfc7643#page-19
```

Comment:
Would have this inline, above. Also point to a section, not a page.

Reply:
done